config/minor_filter.txt

```diff
@@ -41,4 +41,5 @@ hbu=how about you
 \bur\b=your
 btw=by the way
 pls=please
-plz=please
\ No newline at end of file
+plz=please
+tf=the fuck
```

Comment:
```suggestion
\btf\b=the fuck
```

I double checked, Molti was right. This should make it not turn "outfit" into "outhe fuckit"